base/version.jl

```diff
@@ -12,9 +12,12 @@ Version number type which follows the specifications of
 [semantic versioning (semver)](https://semver.org/spec/v2.0.0-rc.2.html), composed of major, minor
 and patch numeric values, followed by pre-release and build
 alphanumeric annotations.
+(As extension to this standard, Julia also allows a single, empty prerelease annotation
+if there is no build identifier present (e.g. `1.0-`), or a single, empty build annotation (e.g. `1.0+`).
 
 `VersionNumber` objects can be compared with all of the standard comparison
 operators (`==`, `<`, `<=`, etc.), with the result following semver v2.0.0-rc.2 rules.
+(However, build annotations are not ignored when comparing version numbers.)
```

Comment:
What does this "However" tell me? Some possible interpretations:
1. semver v2.0.0-rc.2 rules do not specify what to do with build annotations, and we choose to not ignore them, i.e., to take them into account"
   - Then be explicit: "As an extension, build annotations are also taken into account when comparing version numbers." or so
2. semver v2.0.0-rc.2 rules specify that build annotations should be ignored but we do not
   - then we perhaps we should also change the sentence before and write "... with the result following semver ... rules, except that build annotation are *not* ignore ...
3. something else?